arabic/dream/another-dream/It was a nice dream.md

```diff
@@ -0,0 +1 @@
+كنت أحاول الفرار من بعض المجرمين وبينما أركض تسائلت لماذا لا أواجه هؤلاء الأوغاد فاستدرت وواجهتهم حتى قضيت على أربعة رجال فى نصف دقيقة قبل أن ينادينى أحدهم بأن أسرع لأن ميعاد المدرسة سيفوتنى ان لم أستيقظ :) [nice dream](../another dream/It was a nice dream.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.